### PR TITLE
Navbar component

### PR DESCRIPTION
**Pull request by Trigur**
### Changes
Minor refactoring for navigation components

----
UUID: f6a74dee-f832-4860-8f9c-63c2fd26ab43

### CODE REVIEW

**Comment by paglias** [comment]
Hi @Trigur  , sorry for the delay but I'm still catching up from the end of year holidays, I hope to get to this PR in the next couple of days

File: website/client/src/components/challenges/index.vue [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
 <script>
-import SecondaryMenu from '@/components/secondaryMenu';
+import Navbar from '@/components/ui/simpleNavbar';
 
 export default {
   components: {
-    SecondaryMenu,
+    Navbar,
+  },
+  computed: {
```
Comment (paglias) [inline website/client/src/components/challenges/index.vue:21]: I think this can be a normal data property instead of a computed one because the data is static

File: website/client/src/components/groups/index.vue [hunk 2]
```diff
@@ -32,13 +13,32 @@
...
 
 export default {
   components: {
-    groupFormModal,
-    SecondaryMenu,
+    GroupFormModal,
+    Navbar,
+  },
+  computed: {
+    tabs () {
```
Comment (paglias) [inline website/client/src/components/groups/index.vue:25]: doesn't have to a computed prop

File: website/client/src/components/groups/membersModal.vue [hunk 3]
```diff
@@ -10,7 +10,6 @@
       id="members-modal"
       :title="$t('createGuild')"
       size="md"
-      :hide-footer="true"
```
Comment (paglias) [inline website/client/src/components/groups/membersModal.vue:13]: why was this removed?

File: website/client/src/components/groups/membersModal.vue [hunk 4]
```diff
@@ -234,7 +220,7 @@
           </div>
         </div>
       </div>
-      <div class="modal-footer">
+      <div slot="modal-footer">
```
Comment (paglias) [inline website/client/src/components/groups/membersModal.vue:223]: I think this might break the layout of the modal

File: website/client/src/components/hall/index.vue [hunk 5]
```diff
@@ -1,34 +1,37 @@
...
-import SecondaryMenu from '@/components/secondaryMenu';
+import Navbar from '@/components/ui/simpleNavbar';
 
 export default {
   components: {
-    SecondaryMenu,
+    Navbar,
+  },
+  computed: {
+    tabs () {
```
Comment (paglias) [inline website/client/src/components/hall/index.vue:22]: data prop

File: website/client/src/components/inventory/equipment/index.vue [hunk 6]
```diff
@@ -442,9 +430,9 @@ export default {
     ) === CONSTANTS.equipmentDrawerTabValues.COSTUME_TAB;
   },
   methods: {
-    selectDrawerTab (tabName) {
+    selectDrawerTab (index) {
```
Comment (paglias) [inline website/client/src/components/inventory/equipment/index.vue:433]: why this change?

File: website/client/src/components/inventory/index.vue [hunk 7]
```diff
@@ -1,35 +1,41 @@
...
-import SecondaryMenu from '@/components/secondaryMenu';
+import Navbar from '@/components/ui/simpleNavbar';
 
 export default {
   components: {
-    SecondaryMenu,
+    Navbar,
+  },
+  computed: {
+    tabs () {
```
Comment (paglias) [inline website/client/src/components/inventory/index.vue:22]: data prop

File: website/client/src/components/settings/index.vue [hunk 8]
```diff
@@ -1,62 +1,53 @@
...
-import SecondaryMenu from '@/components/secondaryMenu';
+import Navbar from '@/components/ui/simpleNavbar';
 
 export default {
   components: {
-    SecondaryMenu,
+    Navbar,
+  },
+  computed: {
+    tabs () {
```
Comment (paglias) [inline website/client/src/components/settings/index.vue:22]: data prop

**Review by paglias — CHANGES_REQUESTED** [review]
Left some comments @Trigur but we definitely appreciate abstracting some components in the codebase :)

**Comment by Trigur** [comment]
oh, my english very bad...

File: website/client/src/components/groups/membersModal.vue (hunk 3 above)
Comment (Trigur) [reply website/client/src/components/groups/membersModal.vue:13]: because i change class="modal-footer" to slot. And it work's fine
with class:
Снимок экрана 2020-01-19 в 15 48 19
with slot:
Снимок экрана 2020-01-19 в 15 54 10

File: website/client/src/components/hall/index.vue (hunk 5 above)
Comment (Trigur) [reply website/client/src/components/hall/index.vue:22]: i think computed property better for few reasons:
- you may need to add conditions of show for some tabs;
- you can incapsulate this property in mixin later;
- i don't want to allow somebody to add tab in method - it's bad practice (i think);

File: website/client/src/components/inventory/equipment/index.vue (hunk 6 above)
Comment (Trigur) [reply website/client/src/components/inventory/equipment/index.vue:433]: i didn't want to change how drawerHeaderTabs works

File: website/client/src/components/hall/index.vue (hunk 5 above)
Comment (paglias) [reply website/client/src/components/hall/index.vue:22]: That's true @Trigur but right now there are no conditions, if we ever add them we can move the prop to a computed later.

> i don't want to allow somebody to add tab in method - it's bad practice (i think);

that's right! methods should be avoided when only used to calculate some data

File: website/client/src/components/groups/membersModal.vue (hunk 3 above)
Comment (paglias) [reply website/client/src/components/groups/membersModal.vue:13]: thanks, that makes sense!

**Comment by paglias** [comment]
No worries @Trigur , I'd still like the computed props that don't use anything dynamic to be transformed into normal data props

Sorry for the delay!

**Comment by Trigur** [comment]
y y, sure. Waste of time. Gl with project.
